filters/diag/absorb.go

```diff
@@ -82,25 +90,12 @@ func (a absorb) Request(ctx filters.FilterContext) {
 		a.logger.Infof("received request to be absorbed: %s", id)
 	}
 
-	var count = 0
-	buf := make([]byte, 1<<12)
-	for {
-		n, err := req.Body.Read(buf)
-		count += n
-		if !a.silent {
-			a.logger.Infof("request %s, consumed bytes: %d", id, count)
-		}
+	count, err := io.Copy(ioutil.Discard, req.Body)
```

Comment:
this simplification would remove the visibility of slow streams. Let's put back the partial logs for the !silent absorb

Reply:
I've stumbled upon absorb name issue while testing megabyte-sized requests, for them such logging was excessive. The indication of the slow request could be above `received request to be absorbed: %s` and absent subsequent `request finished:`.

For the usecase where one needs to consume request fast this solution is swift while existing is slow.

Reply:
I see. Then maybe we would log the absorbed chunks only with a time threshold, like if now.Since(timeOfLastLog) > time.Second then log and set timeOfLastLog to now. There were some cases when we were investigating how the incoming body is being streamed.

For fast absorb, we still have the absorbSilent, where could use of course io.Copy

Reply:
I've updated PR as suggested